pkg/webui/lib/components/date-time/relative.js

```diff
@@ -0,0 +1,64 @@
+// Copyright © 2019 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { FormattedRelative } from 'react-intl'
+
+import PropTypes from '../../prop-types'
+
+const RelativeTime = function (props) {
+  const {
+    value,
+    updateInterval,
+    options,
+  } = props
+
+  let dateValue = value
+  if (!(value instanceof Date)) {
+    dateValue = new Date(value)
+  }
+
+  const formattedDateTime = dateValue.toISOString()
+
+  return (
+    <FormattedRelative
+      value={value}
+      updateInterval={updateInterval}
+      {...options}
+    >
+      {formattedRelativeTime => (
+        <time dateTime={formattedDateTime}>{formattedRelativeTime}</time>
```

Comment:
```suggestion
        <time title={formattedDateTime} dateTime={formattedDateTime}>{formattedRelativeTime}</time>
```
The relative time element should show the absolute time on hover. As far as I know the `<time>` element does not do this by default. Might be that `<FormattedRelative />` does this though.